tests/unit_and_functional/test_cli.py

```diff
@@ -371,6 +371,12 @@ def test_debug_subcommand_sleeps():
     assert process.returncode == 0
 
 
+def test_configure_cluster_subcommand_exists():
```

Comment:
I wonder how best to test this functionality, given that we can only configure our test cluster once 🤔

Maybe we tear-down the configuration after tests have run and require `configure-cluster` for every integration test?

Reply:
I think the best approach will be to set it up once for all the tests in a test fixture setup. It doesnt make sense to spin this up every time as its stateless and none of our test should modify any aspect of it. Also it will make all tests run longer and eat into our CICD credit. 

Right now the setup namespace tests cover this and with the next ticket where we remove the namespace from other functions it will be tested even more.

Reply:
Agreed.